samples/unit-tests/tooltip/outside/demo.js

```diff
@@ -82,6 +82,68 @@ QUnit.test('Outside tooltip styling', function (assert) {
     );
 });
 
+QUnit.test('Tooltip with default positioner and set outside true on correct position (#16944)', assert => {
+    const chart = Highcharts.chart('container', {
+            chart: {
+                width: 400,
+                height: 400
+            },
+            tooltip: {
+                padding: 0,
+                borderWidth: 0
+            },
+            series: [{
+                data: [5, 3, 4, 7, 2]
+            }]
+        }),
+        point = chart.series[0].points[1],
+        tooltip = chart.tooltip;
+
+    tooltip.refresh(point);
+
+    assert.strictEqual(
+        Math.round(chart.plotLeft + point.plotX - tooltip.label.width / 2),
+        Math.round(tooltip.now.anchorX - tooltip.label.width / 2),
+        'Tooltip position should appear at point with default positioner'
+    );
+
+    chart.update({
+        tooltip: {
+            outside: true
+        }
+    });
+
+    tooltip.refresh(point);
+
+    const tooltipAbsolute = tooltip.now.x + tooltip.now.anchorX,
+        pointerLeft = chart.pointer.getChartPosition().left,
+        pointX = point.plotX + chart.plotLeft + pointerLeft;
+
+    assert.strictEqual(
+        Math.round(tooltipAbsolute),
+        Math.round(pointX),
+        'Tooltip position should appear at point with outside true'
+    );
+
+    chart.update({
+        chart: {
+            margin: 50
+        }
+    });
+
+    tooltip.refresh(point);
+
+    chart.container.style.margin = '200px';
+
+    tooltip.refresh(point);
```

Comment:
Do you need to refresh the tooltip twice?

Reply:
Thanks for a check! I don't need a second refresh.